Code/Engine/RendererVulkan/Cache/Implementation/ResourceCacheVulkan.cpp

```diff
@@ -0,0 +1,320 @@
+#include <RendererVulkan/RendererVulkanPCH.h>
+
+#include <Foundation/Basics.h>
+#include <RendererFoundation/Resources/Texture.h>
+#include <RendererVulkan/Cache/ResourceCacheVulkan.h>
+#include <RendererVulkan/Utils/ConversionUtilsVulkan.h>
+
+ezGALDeviceVulkan* ezResourceCacheVulkan::s_pDevice;
+vk::Device ezResourceCacheVulkan::s_device;
+ezHashTable<ezResourceCacheVulkan::RenderPassDesc, vk::RenderPass, ezResourceCacheVulkan::ezRenderPassHash> ezResourceCacheVulkan::s_renderPasses;
```

Comment:
I think it will be faster to directly use ezGALRenderingSetup as key and then always have a render pass and a framebuffer together in one entry. Or is there any reason why you want to separate those?

Reply:
ezGALDevice::ReplaceExisitingNativeObject prevents hashing of ezGALRenderingSetup as the handles will point to different things every frame. Will try to remove that functionality in my next batch of changes.